platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -758,9 +776,6 @@ private static void updateLoadedStateAddressBook(
         new MerkleRouteIterator(state, state.getPlatformState().getAddressBook().getRoute())
```

Comment:
Is it still necessary to invalidate the hash down to the address book?

Reply:
You are correct, not necessary. These nodes will be unhashed in this method.